Documentation/CRDs/Cluster/external-cluster/upgrade-external.md

```diff
@@ -0,0 +1,53 @@
+# Upgrade external cluster
+
```

Comment:
Could you provide an overview about the upgrades? It will help to understand what type of upgrade is being referred to, instead of only directly diving into details about caps.

Reply:
Still planning on this or is it somewhere else? I was thinking it would be before this heading